packages/xo-web/src/common/intl/messages.js

```diff
@@ -1066,6 +1071,8 @@ const messages = {
   hostRemoteSyslog: 'Remote syslog',
   hostIommu: 'IOMMU',
   hostNoCertificateInstalled: 'No certificates installed on this host',
+  onlyAvailableXcp83OrHighter: 'Only available for XCP-ng 8.3.0 or highter',
```

Comment:
I'm not sure in which order PRs will be merged but keep in mind that the typo is here too :)